controllers/drpolicy_controller_test.go

```diff
@@ -289,12 +288,12 @@ var _ = Describe("DrpolicyController", func() {
 						),
 					},
 				)
-			}()
+			}
 			drp := drpolicy.DeepCopy()
 			drp.Spec.SchedulingInterval = `3s`
-			Expect(k8sClient.Create(context.TODO(), drp)).To(MatchError(err))
+			Expect(k8sClient.Create(context.TODO(), drp)).To(MatchError(err(drp.Spec.SchedulingInterval)))
 			drp.Spec.SchedulingInterval = `0`
-			Expect(k8sClient.Create(context.TODO(), drp)).To(MatchError(err))
+			Expect(k8sClient.Create(context.TODO(), drp)).To(MatchError(err(drp.Spec.SchedulingInterval)))
```

Comment:
Is this related to the envtest change?

Reply:
Yes, as we get a later envtest these fail when run in the latest environment. (same for the other comment)